cwms-data-api/src/main/java/cwms/cda/data/dto/forecast/ForecastSpec.java

```diff
@@ -0,0 +1,30 @@
+package cwms.cda.data.dto.forecast;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement(name = "alias")
+@XmlAccessorType(XmlAccessType.FIELD)
+public class ForecastSpec {

```

Comment:
Shouldn't reference the codes. There should be an officeId, forecastSpecId, sourceEntityId, locationId

If the location office can differ from the forecastSpec office, we'd need an additional locationOfficeId, but I think that's likely not necessary.

Reply:
No, I think maybe a forecast could include data from different offices (for example if raw data is always under "CWMS" or something that could happen a lot) but a forecast itself would I think also be per office.

... wait, oh dear, does a need to always have data from one location? wouldn't the CAVI forecasts be multiple locations?

Reply:
I've got some questions out and another comment about in on the schema cwms_fsct PR.

Reply:
yeah, Peter and I weren't sure about how the location code would actually be used. I presume all location's defined in the time series id's would be relevant.